avm/res/dashboard/grafana/types/grafana.type.bicep

```diff
@@ -0,0 +1,153 @@
+@export()
+type grafanaConfigurationsType = {
+  smtp: smtpType?
+}
+
+@export()
+type smtpType = {
+  enabled: bool
+  fromAddress: string
+  fromName: string
+  host: string
+  password: string
+  skipVerify: bool
+  user: string
+  startTLSPolicy: string
+}
+
+@export()
+type enterpriseConfigurationsType = {
+  marketplaceAutoRenew: string?
+  marketplacePlanId: string?
+}
+
+@export()
+type grafanaIntegrationsType = {
+  azureMonitorWorkspaceIntegrations: azureMonitorWorkspaceIntegrationsType[]?
+}
+
+@export()
+type azureMonitorWorkspaceIntegrationsType = {
+  azureMonitorWorkspaceResourceId: string
+}
+
+@export()
+type roleAssignmentType = {
```

Comment:
Outdated type. Please update to latest (both here as well as in `main.bicep` file